x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_details_page/components/agent_details/agent_details_integrations.tsx

```diff
@@ -107,6 +107,9 @@ export const AgentDetailsIntegration: React.FunctionComponent<{
                 logQuery: getLogsQueryByInputType(inputType),
               })}
               iconType="editorAlignLeft"
+              aria-label={i18n.translate('xpack.fleet.agentDetailsIntegrations.viewLogsButton', {
```

Comment:
The aria text is a clone of the tooltip text in both instances, I thought about putting the translated text in a variable but didn't know if it was overkill?

Reply:
Yeah I think that'd likely be overkill. This looks good to me as-is.